mm2src/coins/utxo/utxo_standard.rs

```diff
@@ -1,17 +1,19 @@
 use super::*;
 use crate::init_withdraw::{InitWithdrawCoin, WithdrawTaskHandle};
-use crate::{CanRefundHtlc, CoinBalance, NegotiateSwapContractAddrErr, SwapOps, TradePreimageValue,
-            ValidateAddressResult, WithdrawFut};
+use crate::{CanRefundHtlc, CoinBalance, GetRawTransactionError, NegotiateSwapContractAddrErr, SwapOps,
+            TradePreimageValue, ValidateAddressResult, WithdrawFut};
+use bitcoin_hashes::hex::ToHex;
 use common::mm_metrics::MetricsArc;
 use common::mm_number::MmNumber;
 use crypto::trezor::TrezorCoin;
 use futures::{FutureExt, TryFutureExt};
+use rpc::v1::types::H256 as H256Json;
 use serialization::CoinVariant;
 use utxo_signer::UtxoSignerOps;
 
 #[derive(Clone, Debug)]
 pub struct UtxoStandardCoin {
-    utxo_arc: UtxoArc,
+    pub(crate) utxo_arc: UtxoArc,
```

Comment:
Is it correct that `pub(crate)` is not required?

Reply:
No, it's not needed so it's better to keep it private.